packages/node/src/sdk.ts

```diff
@@ -149,7 +156,20 @@ export async function close(timeout?: number): Promise<boolean> {
 }
 
 /**
- * A function that returns a Sentry release string dynamically from env variables
+ * Function that checks if autoSessionTracking option is enabled
+ */
+export function isAutosessionTrackingEnabled(): boolean {
```

Comment:
```suggestion
export function isAutoSessionTrackingEnabled(): boolean {
```

The option is called `autoSessionTracking`, so we could as well be consistent with the naming here. This would need an update on the call sites as well.